metrics/metrics.go

```diff
@@ -0,0 +1,177 @@
+package metrics
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"time"
+
+	"github.com/openfresh/plasma/config"
+	"github.com/pkg/errors"
+	metrics "github.com/rcrowley/go-metrics"
+)
+
+const (
+	Log = "log"
```

Comment:
__[golint]__
* [golint] exported const Log should have comment (or a comment on this block) or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->